### PR TITLE
Remove meta data from add event form.

### PR DESCRIPTION
**Pull request by ObothA**
# Summary of changes
- Create a separate form to add an event without meta data.

# How to test
- Clone the repository and checkout to bug/add-event branch.
- Navigate to the calendar.
- Click Create Event and fill out the form.

image of form

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-router-redux": "^4.0.8",
-    "react-scripts": "^4.0.3",
```
Comment (androidcode33) [inline package.json]: @ObothA lets retainer the old version "react-scripts": "^4.0.3". Remove the package.json in your comment. The packages should not be updated.

**Review by androidcode33 — COMMENTED** [review]
@ObothA fix this item package.json

File: package.json (hunk 1 above)
Comment (ObothA) [reply package.json]: @androidcode33 from a fresh install, react scripts had other packages with which dependencies were clashing. With npm it couldn't run at all, yarn tried to fix the issues but mouse events were disabled making the project unusable. So upgrading react scripts fixed that issue. You can try running a fresh setup of the worship harvest branch to confirm. In light of that information, should we maintain the upgrade or we revert?

Comment (androidcode33) [reply package.json]: @ObothA that was on your machine and someone else won't get the same issues. Just ignore the package.json file

Comment (ObothA) [reply package.json]: Alright.

**Comment by androidcode33** [comment]
@ObothA You left package-lock.json. Remove it as well.

**Comment by ObothA** [comment]
> @ObothA You left package-lock.json. Remove it as well.

Done.

**Comment by ObothA** [comment]
> @ObothA You left package-lock.json. Remove it as well.

Now that I think about it, won't removing it altogether delete the one in the source repo! How about generating a new
 package-lock.json with the correct versions of react-scripts?

**Comment by androidcode33** [comment]
You are supposed to just remove it from what is committed, not deleting the files themselves, what did you do?

**Comment by ObothA** [comment]
> You are supposed to just remove it from what is committed, not deleting the files themselves, what did you do?

@androidcode33 Initially I had deleted from my machine. I have now just removed it. It's no longer available in the files changed.

**Pull request merged**